cmd/embed/main.go

```diff
@@ -51,6 +52,7 @@ func main() {
 	flag.StringVar(&pkg, "package", "", "the package to use, defaults to the dir name")
 	flag.StringVar(&output, "out", "embed.go", "the file to generate")
 	flag.StringVar(&root, "root", "", "The root path for embedding")
+	flag.BoolVar(&web, "web", false, "If true, replace back slash separators in filepaths with forward slashes")
```

Comment:
Would it not be simpler and easier to have embedded paths always use a forward slash and fix up [`loadReferenceImage`](https://github.com/google/gapid/blob/ded000ea2f5ba94cb92f944c69e91da84af23676/test/integration/replay/gles/image.go#L59) to use `path.Join` instead of `filepath.Join`?